lib/shoryuken.rb

```diff
@@ -1,6 +1,13 @@
 require 'yaml'
 require 'json'
 require 'aws-sdk-core'
+begin
+  unless defined?(Aws::SQS)
+    require 'aws-sdk-sqs'
+  end
+rescue LoadError
+  fail "AWS SDK 3 requires aws-sdk-sqs to be installed separately. Please add gem 'aws-sdk-sqs' to your Gemfile"
+end
```

Comment:
@pyromaniac `Aws::CORE_GEM_VERSION` is the one that returns > 3. But I think only by checking if `Aws::SQS` we can infer if 3 or not.